modular_skyrat/modules/borgs/code/robot_upgrade.dm

```diff
@@ -263,7 +263,7 @@
 	)
 
 /obj/item/borg/upgrade/cargo_tele
-	name = "borg cargo teleporter module"
+	name = "Cargo teleporter module"
```

Comment:
we don't capitalize these

Reply:
Ah I meant to capitalize the design, woops